resources/local/chainspec.toml.in

```diff
@@ -39,9 +39,6 @@ locked_funds_period = 15
 round_seigniorage_rate = [15_959, 6_204_824_582_392]
 
 [highway]
-# Beginning of era 0. By this time, a sufficient majority (> 50% + F/2 — see finality_threshold_percent below)
-# of validator nodes must be up and running to start the blockchain.
```

Comment:
Maybe this comment should be added to the genesis timestamp.

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/619/commits/eceb679a094d27db7ca537d62275772087e03755